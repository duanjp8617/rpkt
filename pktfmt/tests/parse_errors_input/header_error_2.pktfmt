packet FakeProto {
    header = [
        field_1 = Field {
            bit = 1, 
        },
        field_2 = Field {
            bit = 13, 
        },
        field_3 = Field {
            bit = 2, 
        }
    ]
}

