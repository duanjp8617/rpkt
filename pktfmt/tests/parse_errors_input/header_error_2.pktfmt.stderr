ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/header_error_2.pktfmt 6:9-15
6 |         field_2 = Field {
            ^^^^^^^
header error 2:
header field field_2 is not correctly aligned to the byte boundaries
